### DISCUSSION TITLE
Smart Sprinkler timer Tuya -  NAS-WV03B

### DISCUSSION THREAD

**Discussion by hexstudy** [Koenkk/zigbee2mqtt#23429] [Devices]
I purchased the product on aliexpress.
The NAS-WV03B code was listed

https://it.aliexpress.com/item/1005007103491235.html

I am recognized as:
Model: TS0601
Vendor: _TZE204_a9ojznj8

I would like to try to make it recognized as the original code but I don't know how to do it.

Thank you

**Comment by maxinthecloud** [comment]
Is there a procedure for that?

**Comment by hexstudy** [comment]
This device is compatible with the last dev version of z2m

It's recognized but I'dont see any parameter to start

But you can see it

https://github.com/Koenkk/zigbee2mqtt/issues/22813
